build/js/main.min.js

```diff
@@ -0,0 +1 @@
:2e3})}).error(function(e){l.create({className:"danger",content:e.message,dismissOnTimeout:!0,dismissOnClick:!0,timeout:2e3})})},a.logout=function(){i.removeToken(),t.$storage.$reset(),e.logout(function(){"home"===o.$current.name?o.reload():o.go("home"),l.create({className:"success",content:"successfully logged out",dismissOnTimeout:!0,dismissOnClick:!0,timeout:2e3})},function(){l.create({className:"danger",content:"failed to logout!",dismissOnTimeout:!0,dismissOnClick:!0,timeout:2e3})})},a.authenticate=function(e){i.authenticate(e).then(function(e){i.setToken(e.data.token),t.$storage.activeUser=e.data.user.username,t.$storage.userId=e.data.user._id,o.go("home"),t.$storage.loggedIn=!0,l.create({className:"success",content:"You are signed in",dismissOnTimeout:!0,dismissOnClick:!0,timeout:2e3})})["catch"](function(e){l.create({className:"danger",content:e.data.message,dismissOnTimeout:!0,dismissOnClick:!0,timeout:2e3})})},a.isAuthenticated=function(){i.isAuthenticated()}}]),angular.module("hairvenApp").value("baseUrl","http://localhost:3030");var app=angular.module("hairvenApp");app.factory("HairstyleService",["$http","baseUrl","$localStorage","Upload",function(e,s,t,r){var a={getAllHairstyles:function(){return e.get(s+"/api/hairstyles/")},getSalonHairstyles:function(t){return e.get(s+"/api/salons/"+t+"/hairstyles")},getOneHairstyle:function(t){return e.get(s+"/api/hairstyles/"+t)},addHairstyle:function(e,t){return r.upload({url:s+"/api/hairstyles",data:{hairPhoto:e,name:t.name,description:t.description,salon:t.salon},method:"POST"})},updateHairstyle:function(t,r){return e.put(s+"/api/hairstyles/"+t,r)},deleteHairstyle:function(t){return e["delete"](s+"/api/hairstyles/"+t)}};return a}]);var app=angular.module("hairvenApp");app.factory("SalonService",["$http","baseUrl","$localStorage",function(e,s,t){var r={getSalon:function(t){return e.get(s+"/api/salons/"+t)},addSalon:function(t){return e.post(s+"/api/salons/",t)},updateSalon:function(t,r){return e.put(s+"/api/salons/"+t,r)},deleteSalon:function(t){return e["delete"](s+"/api/salons/"+t)}};return r}]);var app=angular.module("hairvenApp");app.factory("UserService",["$http","baseUrl","$localStorage",function(e,s,t){function r(e){var s=e.replace("-","+").replace("_","/");switch(s.length%4){case 0:break;case 2:s+="==";break;case 3:s+="=";break;default:throw"Illegal base64url string!"}return window.atob(s)}function a(){var e=t.token,s={};if("undefined"!=typeof e){var a=e.split(".")[1];s=JSON.parse(r(a))}return s}function o(e){angular.extend(n.currentUser,e)}var n=(a(),{register:function(t){return e.post(s+"/api/signup",t)},login:function(t){return e.post(s+"/api/login",t)},facebookLogin:function(){e.get(s+"/api/auth/facebook/")},twitterLogin:function(){e.get(s+"/api/auth/twitter/")},logout:function(e){o({}),e()},currentUser:function(){a()},updateUser:function(t,r,a){e.put(s+"/api/users/"+t).success(r).error(a)},deleteUser:function(t,r,a){e["delete"](s+"/api/users/"+t).success(r).error(a)}});return n}]);
```

Comment:
Line is too long.<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>Expected '{' and instead saw 't'.<br>Expected '{' and instead saw 'throw'.<br>Expected '{' and instead saw 'return'.<br>Identifier 'oauth_token' is not in camel case.<br>Identifier 'oauth_verifier' is not in camel case.<br>Identifier 'consumer_key' is not in camel case.<br>Identifier 'consumer_secret' is not in camel case.<br>Identifier 'screen_name' is not in camel case.<br>Missing '()' invoking a constructor.<br>'o' used out of scope.<br>Identifier 'client_id' is not in camel case.<br>Too many errors. (100% scanned).